ios/FluentUI.Demo/FluentUI.Demo/Demos.swift

```diff
@@ -7,24 +7,24 @@ import UIKit
 
 // Register your control demos here
 let demos: [(title: String, controllerClass: UIViewController.Type)] = [
+    ("Avatar (Vnext)", AvatarDemoController.self),
```

Comment:
Need a formal name instead of Vnext?  "Avatar 2.0" ?

Reply:
The problem with "2.0" is that we're starting with version 0.0.1 for the private drop....
Maybe once we merge it to master this could be a good strategy: "Avatar 0.3" or something like that.